pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -64,15 +65,20 @@ type provider interface {
 	OutboundDispatcher() dispatcher.Outbound
 }
 
+type connectionRecorder interface {
```

Comment:
Record retrieval doesn't seem like a "Recorder"

Reply:
Rename to connectionStore